collection_manager/containers/docker/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+python /collection_manager/collection_manager/main.py \
```

Comment:
The setup.py file can also generate a command line this first line shorted (for example collection-manager):

entry_points={
        'console_scripts': ['collection-manager=collection_manager.collection_manager:main']
    }

Reply:
No sure if this is relevant, you will see.

Reply:
That's cool, I didn't know about that!